doc/howto/usage/cluster/k8s-aws/README.md

```diff
@@ -0,0 +1,650 @@
+#PaddlePaddle on AWS with Kubernetes
+
+##Prerequisites
+
+First, you need an AWS account, please check out [this](http://docs.aws.amazon.com/lambda/latest/dg/setting-up.html) for how to setup an AWS account. 
```

Comment:
please follow [this guide](http://docs.aws.amazon.com/lambda/latest/dg/setting-up.html) to set up your AWS account.